src/presenter/presenter.js

```diff
@@ -0,0 +1,22 @@
+import AddPointForm from '../view/add-form-view';
+import EditForm from '../view/edit-form-view';
+import FiltersForm from '../view/filters-view';
+import SortForm from '../view/sorts-view';
+import TripPoint from '../view/trip-point-view';
+import { render } from '../render';
+import { POINTS_COUNT } from '../const';
+
+export default class Presentor {
+  init = (containers) => {
+    const {controlsContainer, eventsContainer} = containers;
+    render(new FiltersForm(), controlsContainer);
+    render(new SortForm(), eventsContainer);
+    render(new EditForm(), eventsContainer);
+    render(new AddPointForm(), eventsContainer);
+    let count = 0;
+    while (count < POINTS_COUNT) {
+      render(new TripPoint(), eventsContainer);
+      count++;
+    }
```

Comment:
придумывание имени переменной (или функции) вещь сложная. Особенно, если переменная выполняет вспомогательные действия. Это одна из причин, которая вынудила нас договорится, что там где можно не использовать временные переменные. Пусть компилятор ломает над этим голову.

```suggestion
    Array.from({length: POINTS_COUNT}, ()=>POINTS_COUNT)
```

Обрати внимание, что слово length мы тут не выдумываем - оно уже было выдумано до нас.

Reply:
согласен,  Array.from() - красивая штука !